app/assets/stylesheets/articles.scss

```diff
@@ -1564,6 +1565,7 @@
   animation-timing-function: linear;
   background: #f6f6f6;
   background: linear-gradient(to right, #f6f6f6 8%, #f0f0f0 18%, #f6f6f6 33%);
+  background: var(--theme-gradient-background, linear-gradient(to right, #f6f6f6 8%, #f0f0f0 18%, #f6f6f6 33%));
```

Comment:
We should add new variables to #1377 or start the documentation page.

Reply:
Yeah, a doc page makes sense.